api/src/main/java/org/openmrs/module/referencemetadata/OrderFrequencyMetaData.java

```diff
@@ -0,0 +1,198 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.referencemetadata;
+
+import org.openmrs.Concept;
+import org.openmrs.OrderFrequency;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.metadatadeploy.bundle.AbstractMetadataBundle;
+import org.springframework.stereotype.Component;
+
+/**
+ * This class contains the logic for adding the best practice order frequencies into the order_frequency database table
+ */
+@Component("referenceApplicationOrderFrequencies")
+public class OrderFrequencyMetaData extends AbstractMetadataBundle {
+
+	/**
+	 * This method contains the logic for creating the orderFrequency object needed to populate the order_frequency table
```

Comment:
Logic for creating? Shouldn't you just say it creates...?

Reply:
Okay. Thanks for pointing this out